### DISCUSSION TITLE
"There is no server available!"

### DISCUSSION THREAD

**Discussion by jake7323** [ImRidingNow/PavlovRconWebserver#89] [Q&A]
Hello,

Thanks for the work, I have followed the Pavlov wiki to create a server on an Ubuntu server VM and I'm attempting to get your rcon web server to work. I have used the docker approach you provide on the same ubuntu vm and I can reach the web server via the http://x.x.x.x:5001, but it shows "there is no server available!" what must I do to correct this? I have RconSettings.txt in my pavlov server config folder. What port needs to be used there?

Thanks!

**Comment by ImRidingNow** [comment]
We do discuss that on the discord :)
